Sources/DequeModule/Deque+Extras.swift

```diff
@@ -166,6 +166,12 @@ extension Deque {
   /// - SeeAlso: `append(contentsOf:)`
   @inlinable
   public mutating func prepend<S: Sequence>(contentsOf newElements: S) where S.Element == Element {
+    let done: Void? = newElements._withContiguousStorageIfAvailable_SR14663 { source in
```

Comment:
I couldn't resist slipping in this trivial performance improvement, if only for symmetry with the sequence-based `append(contentsOf:)`.